src/date-range-picker/__integ__/events.test.ts

```diff
@@ -24,73 +24,96 @@ class DateRangePickerEventPage extends DateRangePickerPage {
   }
 }
 
-describe.each<boolean>([false, true])('DatePicker blur events (expandTOViewport=%s)', expandToViewport => {
-  const setupTest = (testFn: (page: DateRangePickerEventPage) => Promise<void>) => {
+describe('Date Range Picker', () => {
+  const setupTest = (
+    testFn: (page: DateRangePickerEventPage) => Promise<void>,
+    granularity: 'day' | 'month',
+    expandToViewport: boolean
+  ) => {
     return useBrowser(async browser => {
       const page = new DateRangePickerEventPage(
         createWrapper().findDateRangePicker().getElement(),
         browser,
         expandToViewport
       );
-      await browser.url(`#/light/date-range-picker/with-event-handlers?expandToViewport=${expandToViewport}`);
+      const params = new URLSearchParams({
+        monthOnly: granularity === 'month' ? 'true' : 'false',
+        expandToViewport: `${expandToViewport}`,
+      });
+      await browser.url(`#/light/date-range-picker/with-event-handlers?${params}`);
       await page.waitForLoad();
       await testFn(page);
     });
   };
 
-  test(
-    'onBlur is fired when clicking next to the date range picker',
-    setupTest(async page => {
-      await page.focusTrigger();
-      await expect(page.getText('#onFocusEvent')).resolves.toBe('onFocus event called 1 times.');
-      await page.keys('Enter');
-      await expect(page.isDropdownOpen()).resolves.toBe(true);
-      await expect(page.isDropdownFocused()).resolves.toBe(true);
+  describe.each<boolean>([false, true] as const)('DatePicker blur events (expandTOViewport=%s)', expandToViewport => {
```

Comment:
nit: I think the `<boolean>` and `as const` annotations are not necessary here as the boolean type should be correctly inferred

Reply:
nit2: there is a small typo in expandTOViewport, should be expandToViewport